Makefile

```diff
@@ -80,13 +89,21 @@ shellcheck: ## Lints Bash and sh scripts.
 # don't maintain those scripts. Omitting the `.venv/` dir because we don't control
 # files in there. Omitting the ossec packages because there are a LOT of violations,
 # and we have a separate issue dedicated to cleaning those up.
+	-@docker rm -f shellcheck-targets
```

Comment:
this line generates a kind of confusing error message so maybe it should be eliminated.

https://circleci.com/gh/freedomofpress/securedrop/4043#build-parameters/containers/0 for an example.